ambry-frontend/src/main/java/com.github.ambry.frontend/AmbryBlobStorageService.java

```diff
@@ -55,8 +56,8 @@
  * All the operations that need to be performed by the Ambry frontend are supported here.
  */
 class AmbryBlobStorageService implements BlobStorageService {
-  protected static final ByteBuffer EMPTY_BUFFER = ByteBuffer.allocate(0);
```

Comment:
I don't think it was - not sure why this variable existed and was private. Might have been an oversight during a refactoring.